app/src/NativeBase/Components/Forms/ResponseModal.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file Modal shows response from api.
+ */
+
+import React, { FC } from 'react'
+import { Text, Modal, Icon } from 'native-base'
+import { Pressable } from 'react-native'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+interface ResponseModalProps {
+  message: string
+  header: string
+  success: boolean
+  isOpen: boolean
+  onClose: (value: boolean) => void
+}
+
+const ResponseModal: FC<ResponseModalProps> = ({
+  message,
+  header,
+  success,
+  isOpen,
+  onClose,
+}) => {
+  if (success) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        onClose={onClose}
+        justifyContent="center"
+        size="xl"
+      >
+        <Modal.Content>
+          <Pressable onPress={() => onClose(true)}>
+            <Modal.Header
+              alignItems="center"
+              borderBottomWidth="0"
+              pb="0"
+              _dark={{ color: 'textDarkMode.100' }}
+              _light={{ color: 'text.100' }}
+            >
+              <Icon
+                as={MaterialIcons}
+                mb="4"
```

Comment:
Tiny thing, but you can use a shorthand of `my` (or `marginY`) to set both top and bottom margins to the same value, instead of having to specify each